Sources/SwiftSyntax/SourceLocation.swift

```diff
@@ -511,21 +481,21 @@ extension SyntaxProtocol {
 /// the end-of-file position.
 fileprivate func computeLines(
   tree: Syntax
-) -> ([AbsolutePosition], AbsolutePosition) {
-  var lines: [AbsolutePosition] = []
-  // First line starts from the beginning.
-  lines.append(.startOfFile)
+) -> (
+  lines: [AbsolutePosition],
+  endOfFile: AbsolutePosition,
+  sourceLocationDirectives: [(sourceLine: Int, arguments: SourceLocationDirectiveArguments?)]
+) {
+  var lines: [AbsolutePosition] = [.startOfFile]
   var position: AbsolutePosition = .startOfFile
-  let addLine = { (lineLength: SourceLength) in
+  var sourceLocationDirectives: [(sourceLine: Int, arguments: SourceLocationDirectiveArguments?)] = []
+  let prefix = tree.raw.forEachLineLength { lineLength in
```

Comment:
`prefix` doesn’t really make sense as a variable name here, right? Shouldn’t it be `sourceFileLength` or something? Also, the usage of `prefix` in `RawSyntax.forEachLineLength` + the comment on it seem pretty confusing to me. Not your doing but might be worth to clean that up along the way.

Reply:
This `prefix` is more like `lastLineLength`. I changed the naming so.
I didn't changed the naming in `forEachLineLength` functions, because I wasn't able to come up with better naming. It's like "the length of the current line _so far_"